dev/arxiv_category_converter.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
```

Comment:
Is this script meant to be **imported purely as a module**? If it is, I think it is not needed here

Reply:
I mean the shebang